### ISSUE TITLE
performance comparisons accidentally compare apples and oranges

### ISSUE THREAD

**Issue by adriancole** [open-telemetry/opentelemetry-collector#355]
If someone reads this without thinking, they'll jump to conclusion "zipkin is slow"

https://github.com/open-telemetry/opentelemetry-service/blob/master/docs/performance.md#results-without-tail-based-sampling

Why are we not comparing apples and apples? Ex binary format with binary format, or text vs text? Zipkin's http endpoint has accepted protobuf since before this project started

Is there some difficulty encoding protobuf? If not, can you please correct this?

https://github.com/openzipkin/zipkin/releases/tag/2.8.3

**Comment by tigrannajaryan** [comment]
Zipkin exporter and performance measurement results are inherited from OpenCensus Service. I added a separate issue to add binary format to Zipkip exporter: https://github.com/open-telemetry/opentelemetry-service/issues/356
Once that is done we can update performance results.

**Comment by adriancole** [comment]
thanks @tigrannajaryan looking forward to it.

**Comment by bogdandrutu** [comment]
The performance doc is outdated, closing this in favor of https://github.com/open-telemetry/opentelemetry-collector/issues/1450
